config/routes.rb

```diff
@@ -195,6 +192,12 @@
     get "admin", to: "admin/dashboards/summary#show"
 
     namespace :admin do
+      namespace :settings do
+        resources :webhook_messages, only: %i[index show] do
+          resources :processing_jobs, only: %i[create], controller: "webhook_messages/processing_jobs"
```

Comment:
```suggestion
        resources :webhook_messages, only: %i[index show], path: "webhook-messages" do
          resources :processing_jobs, only: %i[create], controller: "webhook_messages/processing_jobs", path: "processing-jobs"
```
just for consistency - I think all paths should use hyphens rather than underscores.